tests/unit/test_utils_general.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+from pathlib import Path
+from schematic.utils.general import (
+    check_synapse_cache_size,
+)
+
+import pytest
+from unittest.mock import MagicMock
+from unittest import mock
+
+
+@pytest.mark.parametrize(
```

Comment:
Need to move tests from test_utils.py

Reply:
Done here: https://github.com/Sage-Bionetworks/schematic/pull/1542/commits/ffa9498fb55b602182beb28e60210734a24a8bbd